.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+- package-ecosystem: mix
+  directory: "/"
+  schedule:
+    interval: daily
```

Comment:
I think we can make the interval weekly. We have a lot of notifications for updating the dependencies on the repositories and this creates a lot of noise. But happy to keep it daily on specific repositories too.

Reply:
Changed 👍

Reply:
I'm still not sure if this should be daily or weekly for this repo. While this repo is in active development we might want to keep it daily, I was more thinking of other "stable" repos.
We can change this back on the next PRs if required

Reply:
I'll leave this open for @nelsonic to comment. I'll change it back to daily, meanwhile.

Reply:
Totally happy with Weekly for the demo/example. Agree that it's noisy. Let's start changing the frequency gradually on all _those_ our repos. However on the `App` and `Auth` we want _daily_ to for security updates. 💭